libarchive/archive_string.c

```diff
@@ -1526,7 +1528,7 @@ get_current_codepage(void)
 	p = strrchr(locale, '.');
 	if (p == NULL)
 		return (GetACP());
-	if (strcmp(p+1, "utf8") == 0)
+	if ((strcmp(p+1, "utf8") == 0) || (strcmp(p+1, "UTF-8") == 0))
```

Comment:
Idk if this is a result of CRT behavior changing over time, but this always seems to be `UTF-8`. It may also be a decent idea to do a case-insensitive check here.